spec/outputs/lumberjack_spec.rb

```diff
@@ -1 +1,89 @@
+# encoding: utf-8
+require "logstash/outputs/Lumberjack"
```

Comment:
Typo here, specs don't pass because of this.

```
LoadError: no such file to load -- logstash/outputs/Lumberjack
          require at org/jruby/RubyKernel.java:1040
           (root) at /home/jls/projects/logstash-output-lumberjack/spec/outputs/lumberjack_spec.rb:2

```